static/src/javascripts/projects/common/modules/sport/score-board.js

```diff
@@ -44,7 +46,10 @@ define([
     ScoreBoard.prototype.componentClass = 'match-summary';
 
     ScoreBoard.prototype.prerender = function () {
-        this.placeholder.innerHTML = '';
+        var scoreLoadingPlaceholder = $('.score__loading', $(this.placeholder));
+        if (scoreLoadingPlaceholder.length) {
+            scoreLoadingPlaceholder.remove();
+        }
```

Comment:
Might be worth documenting this in the code, so future maintainers don't get confused!